pom.xml

```diff
@@ -4,14 +4,27 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.596.1</version>
+        <version>3.50</version>
```

Comment:
If you update now, it might be better to go with the recent 4.0 release. https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.0